ipatests/test_integration/test_installation.py

```diff
@@ -1108,6 +1115,79 @@ def run_request(url, expected_stdout=None, expected_stderr=None):
                 expected_stdout=f'href="https://{self.master.hostname}/'
             )
 
+    @master_unistalled
+    def test_hostname_parameter(self):
+        """
+        Test for #2692 ipa-server-install ignores --hostname
```

Comment:
1. issue number is duplicated in the link below, so it is not needed here.
2. Please try to follow [PEP257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings), specifically
> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".
3. The docstring should provide enough information about the purpose of the test. Now I need to follow the link to get what the test is trying to check.

What I mean is that docstring should say what test does, not what the bug was. Something like "Check ipa-server-install respects --hostname option in interactive mode."

Same applies to following tests